modernizer-maven-plugin/src/main/java/org/gaul/modernizer_maven_plugin/ModernizerMojo.java

```diff
@@ -212,6 +212,23 @@ public void execute() throws MojoExecutionException {
             }
         }
 
+        String path = outputDirectory.toString();
+        while (!path.endsWith("target")) {
```

Comment:
Is it possible to share the file logic and reference the file locations as a constant? Could the plugin have the annotation processor as a dependency?

Reply:
I think with the existing structure, it's not possible to add the processor as a dependency to the plugin.